Index: Graphics_SDK_4_05_00_03/GFX_Linux_KM/services4/3rdparty/dc_omapfb3_linux/omaplfb_linux.c
===================================================================
--- Graphics_SDK_4_05_00_03.orig/GFX_Linux_KM/services4/3rdparty/dc_omapfb3_linux/omaplfb_linux.c	2014-02-19 16:18:02.595367020 +0100
+++ Graphics_SDK_4_05_00_03/GFX_Linux_KM/services4/3rdparty/dc_omapfb3_linux/omaplfb_linux.c	2014-02-19 16:21:26.070197569 +0100
@@ -281,6 +281,7 @@
 #endif
 }
 
+#if LINUX_VERSION_CODE <= KERNEL_VERSION(3,1,0)
 OMAPLFB_UPDATE_MODE OMAPLFBGetUpdateMode(OMAPLFB_DEVINFO *psDevInfo)
 {
 	struct omap_dss_device *psDSSDev = fb2display(psDevInfo->psLINFBInfo);
@@ -354,6 +355,7 @@
 	return (res == 0);
 //return 1;
 }
+#endif
 
 OMAPLFB_BOOL OMAPLFBWaitForVSync(OMAPLFB_DEVINFO *psDevInfo)
 {
