diff --git a/drivers/video/omap2/dss/dpi.c b/drivers/video/omap2/dss/dpi.c
index fb6463f..c8bbaab 100644
--- a/drivers/video/omap2/dss/dpi.c
+++ b/drivers/video/omap2/dss/dpi.c
@@ -157,20 +157,25 @@ int omapdss_dpi_display_enable(struct omap_dss_device *dssdev)
 {
 	int r;
 
+	pr_info("DSS display enable: %p\n", dssdev);
 	r = omap_dss_start_device(dssdev);
 	if (r) {
 		DSSERR("failed to start device\n");
 		goto err0;
 	}
 
+	pr_info("DSS reg enable: %p\n", dssdev);
 	if (cpu_is_omap34xx()) {
 		r = regulator_enable(dpi.vdds_dsi_reg);
 		if (r)
 			goto err1;
 	}
 
+	pr_info("DSS clk enable: %p\n", dssdev);
 	dss_clk_enable(DSS_CLK_ICK | DSS_CLK_FCK1);
 
+	pr_info("DSS basic init: %p\n", dssdev);
+
 	r = dpi_basic_init(dssdev);
 	if (r)
 		goto err2;
@@ -181,12 +186,15 @@ int omapdss_dpi_display_enable(struct omap_dss_device *dssdev)
 	if (r)
 		goto err3;
 #endif
+	pr_info("DSS set mode: %p\n", dssdev);
 	r = dpi_set_mode(dssdev);
 	if (r)
 		goto err4;
 
 	mdelay(2);
 
+	pr_info("DSS manager enable: %p\n", dssdev->manager);
+
 	dssdev->manager->enable(dssdev->manager);
 
 	return 0;
