Index: php-5.3.27/acinclude.m4
===================================================================
--- php-5.3.27.orig/acinclude.m4	2014-03-26 18:25:19.131539971 +0100
+++ php-5.3.27/acinclude.m4	2014-03-27 10:06:17.000000000 +0100
@@ -2439,7 +2439,7 @@
   dnl Check libc first if no path is provided in --with-iconv
   dnl
 
-  if test "$PHP_ICONV" = "yes"; then
+  if test "$PHP_ICONV" != "no"; then
     dnl Reset LIBS temporarily as it may have already been included
     dnl -liconv in.
     LIBS_save="$LIBS"
Index: php-5.3.27/ext/iconv/config.m4
===================================================================
--- php-5.3.27.orig/ext/iconv/config.m4	2013-07-10 19:43:08.000000000 +0200
+++ php-5.3.27/ext/iconv/config.m4	2014-03-27 10:09:05.563249609 +0100
@@ -15,7 +15,7 @@
 
   if test "$iconv_avail" != "no"; then
     if test -z "$ICONV_DIR"; then
-      for i in /usr/local /usr; do
+      for i in $PHP_ICONV /usr/local /usr; do
         if test -f "$i/include/iconv.h" || test -f "$i/include/giconv.h"; then
           PHP_ICONV_PREFIX="$i"
           break
